### ISSUE TITLE
Port twisted/persistent/sob.py to have python3 syntax and imports

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7437]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7437|
|Type|enhancement|
|Created|2014-05-29 02:19:41Z|

<details><summary>Searchable metadata</summary>

```
trac-id__7437 7437
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1401329981000000 1401329981000000
changetime__1401357443000000 1401357443000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by https://github.com/twisted/twisted/pull/28

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

Duplicate of [#6910](https://github.com/twisted/twisted/issues/6910) which itself should probably be closed as wontfix due to [#3843](https://github.com/twisted/twisted/issues/3843).
